ydb/_topic_reader/topic_reader_asyncio_test.py

```diff
@@ -1004,6 +1012,42 @@ async def test_receive_batch_nowait(self, stream, stream_reader, partition_sessi
         with pytest.raises(asyncio.QueueEmpty):
             stream.from_client.get_nowait()
 
+    @pytest.mark.parametrize(
+        "stream_reader_started",
+        [
+            (
+                PublicReaderSettings(
+                    consumer="test-consumer",
+                    topic="test-topic",
+                    update_token_interval=0.1,
+                ),
+                lambda: "foo-bar",
+            )
+        ],
+        indirect=True,
+    )
+    async def test_update_token(self, stream, stream_reader_started: ReaderStream):
```

Comment:
you can use stream_reader.

it is bad naming of future.
The stream_reader_started should be renamed for better difference between it.

stream_reader - started, then it close by future and check about it closed without error.

stream_reader_started - started too, but response of close - in test. And no check for close without error.